filebeat/prospector/log/harvester.go

```diff
@@ -141,3 +143,7 @@ func (h *Harvester) forwardEvent(data *util.Data) error {
 
 	return nil
 }
+
+func (h *Harvester) ID() uuid.UUID {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Harvester.ID should have comment or be unexported